src/language/__tests__/printer.js

```diff
@@ -34,6 +34,23 @@ describe('Printer', () => {
     );
   });
 
+  it('correctly prints non-query operations without name', () => {
```

Comment:
This test could potentially be within kitchen sink test, but I felt like I'd slow down the tests using it. Let me know if you think otherwise.